src/app/frontend/common/components/resourcelist/secret/component.ts

```diff
@@ -29,7 +29,6 @@ import {ListGroupIdentifier, ListIdentifier} from '../groupids';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class SecretListComponent extends ResourceListBase<SecretList, Secret> {
-  @Input() title = 'Secrets';
```

Comment:
This can't be removed as it is used:
https://github.com/kubernetes/dashboard/blob/bd025c33bd36e214185d91796d12f6030b191c23/src/app/frontend/resource/cluster/serviceaccount/detail/template.html#L22-L23